docs/File-Formats.md

```diff
@@ -610,20 +610,24 @@ The mutation data file extends the [Mutation Annotation Format](https://docs.gdc
 
 ### Minimal MAF format
-A minimal mutation annotations file can contain just three of the MAF columns plus one annotation column. From this minimal MAF, it is possible to create an extended MAF by running it through the [Genome Nexus Annotation Pipeline](https://github.com/genome-nexus/genome-nexus-annotation-pipeline).
-1. **Hugo_Symbol (Required)**: (MAF column) A [HUGO](https://www.genenames.org/) gene symbol.
-2. **Tumor_Sample_Barcode (Required)**: (MAF column) This is the sample ID as listed in the clinical data file.
-3. **Variant_Classification (Required)**: (MAF column) Translational effect of variant allele. Allowed values (from [Mutation Annotation Format page](https://docs.gdc.cancer.gov/Data/File_Formats/MAF_Format/)): `Frame_Shift_Del, Frame_Shift_Ins, In_Frame_Del, In_Frame_Ins, Missense_Mutation, Nonsense_Mutation, Silent, Splice_Site, Translation_Start_Site, Nonstop_Mutation, 3'UTR, 3'Flank, 5'UTR, 5'Flank, IGR, Intron, RNA, Targeted_Region, De_novo_Start_InFrame, De_novo_Start_OutOfFrame`. cBioPortal skips the following types during the import: `Silent, Intron, 3'UTR, 3'Flank, 5'UTR, 5'Flank, IGR and RNA`. Two extra values are allowed by cBioPortal here as well: `Splice_Region, Unknown`. :warning: the values should be in the correct case. E.g. `missense_mutation` is not allowed, while `Missense_Mutation` is.
-4. **HGVSp_Short (Required)**: (annotation column) Amino Acid Change, e.g. p.V600E.
+A minimal mutation annotations file can contain just the six genomic change columns plus one sample identifier column. From this minimal MAF, it is possible to create an extended MAF by running it through the [Genome Nexus Annotation Pipeline](https://github.com/genome-nexus/genome-nexus-annotation-pipeline).
+1. **NCBI_Build (Required)**: The Genome Reference Consortium Build is used by a variant calling software. It must be "GRCh37" or "GRCh38" for a human, and "GRCm38" for a mouse.
+2. **Chromosome (Required)**:  A chromosome number, e.g., "7".
+3. **Start_Position (Required)**: Start position of event.
+4. **End_Position (Required)**: End position of event.
+5. **Reference_Allele (Required)**: The plus strand reference allele at this position.
+6. **Tumor_Seq_Allele2 (Required)**: Primary data genotype.
+7. **Tumor_Sample_Barcode (Required)**: This is the sample ID. Either a TCGA barcode (patient identifier will be extracted), or for non-TCGA data, a literal SAMPLE_ID as listed in the clinical data file.
 
-Next to Hugo_Symbol, it is recommended to have the Entrez gene ID:
+In addition to the above columns, it is recommended to have the read counts to calculate variant allele frequencies:
+
+8. **t_alt_count (Optional, but recommended)**: Variant allele count (tumor). 
+9. **t_ref_count (Optional, but recommended)**: Reference allele count (tumor).
```

Comment:
great that it says recommended 👍

Reply:
It seems like `t_alt_count` and `t_ref_count` are listed twice? They are also listed below